### DISCUSSION TITLE
v1.12.22

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#2514] [分享 / Share]
- 适配新版动态首页的话题, 包括`动态过滤器`, `夜间模式`等. (#2479, PR #2491 by [JLoeve](https://github.com/LonelySteve))
- 修复依赖 `playerReady` 的功能失效问题, 包括`外置稍后再看`, `启用快速收藏`, `传统连播模式`等. (#2406)
> 不过最近偶尔也会刷不出外置稍后再看, 我还没搞懂是因为什么, 总之和 #2406 是两码事. (#2487)
- 修复`简化直播间`开启`标题栏活动`时, 标题高度被截断.
- 恢复番剧区的`视频截图`和`逐帧调整`. (#1992)
- 修复番剧区宽屏 + mini 播放器出现的布局问题. (PR #2371 by [FoundTheWOUT](https://github.com/FoundTheWOUT))
- `自定义顶栏`排除游戏中心. (#2358)
- 修复新版首页`隐藏顶部横幅`样式异常. (#2373)
- 部分视频功能排除风纪委员会内嵌播放器. (#2340)

大概是 v1 最后一次正式版更新

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v1.12.22'>v1.12.22</a>.</em>

**Comment by kethoven** [comment]
v1版本感觉还是比v2版本更加的美观，分类更加清晰。

**Reply by the1812** [reply]
因为分类方式已经改掉了, v2 里是标签.
v1 里组件只能塞一个类别下, 像是`强制保留弹幕发送栏`是在视频播放器上的样式更改, 我放在`样式`下, 在`视频`类别里就找不到.

~~当然, 你给 v2 写一个 v1 版的设置面板换掉原来的也是可以的~~

**Reply by hv0905** [reply]
v2虽然灵活, 但是少了那种开箱即用的感觉, 配置比较麻烦
各有好坏吧

**Reply by RingingVolcano** [reply]
> v2虽然灵活, 但是少了那种开箱即用的感觉, 配置比较麻烦
> 各有好坏吧

V2解耦合还是为了更好地可持续发展

**Reply by RiSeikiqwq** [reply]
0

**Comment by Malus-Mill** [comment]
表示挺喜欢开箱即用的感觉

**Reply by RingingVolcano** [reply]
V2解耦合还是为了更好地可持续发展
